modules/apps/translation/translation-api/src/main/java/com/liferay/translation/translator/Translator.java

```diff
@@ -15,14 +15,20 @@
 package com.liferay.translation.translator;
 
 import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.module.configuration.ConfigurationException;
+
+import org.osgi.annotation.versioning.ProviderType;
 
 /**
  * @author Adolfo Pérez
  */
+@ProviderType
 public interface Translator {
 
 	public boolean isEnabled();
 
+	public boolean isEnabled(long idCompany) throws ConfigurationException;
```

Comment:
Rename parameter to "companyId"; if the parameterless "isEnabled" method is no longer used, deprecate it.

Reply:
As there's no longer a system config. make the parameterless version delegate to the new one, use the companyId in CompanyThreadLocal.

Reply:
deprecated!